### ISSUE TITLE
Tag issues for Hactoberfest?

### ISSUE THREAD

**Issue by Ainali** [govdirectory/website#95]
[Hacktoberfest](https://hacktoberfest.digitalocean.com/) comes up in about a month and could be a good way to get visibility of the project? Perhaps we could have a short session where we create issues and tag existing ones for things that we deem might be suitable for someone to hack on during October?

**Comment by Abbe98** [comment]
Maybe, let's wait for an official announcement, considering last year's event.

The issues would maybe be the same as the ones tagged [good first issue](https://github.com/govdirectory/website/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).

**Comment by Ainali** [comment]
They already have a countdown clock, I think that strongly implies it will happen. Or do you want to know more about the details before committing?

**Comment by Abbe98** [comment]
I don't think the page has been updated since last year(it still says 2020), the clock is just counting to October clientside.

After last year's backslash, I would expect an announcement having information on how it will actually happen.

**Comment by Abbe98** [comment]
Hacktoberfest have now been announced:
https://hacktoberfest.digitalocean.com/

**Comment by Ainali** [comment]
So do we want to participate? If so, I guess that we in a 30-minute session could get a bunch of issues ready.

**Comment by Abbe98** [comment]
Yeah, it would be a good exercise to get more good first issues in! Maybe also some around design?
